tests/unit/s2n_psk_test.c

```diff
@@ -516,6 +516,69 @@ int main(int argc, char **argv)
 
             EXPECT_SUCCESS(s2n_connection_free(conn));
         }
+
+        /* Test s2n_connection_set_psks */
+        {
+            uint8_t identity_0[] = "identity";
+            uint8_t secret_0[] = "secret";
+
+            uint8_t identity_1[] = "identity 1";
+            uint8_t secret_1[] = "secret 1";
+
+            struct s2n_psk_identity first_identity = { identity_0, sizeof(identity_0) };
+            struct s2n_pre_shared_key first_psk = { first_identity, secret_0, sizeof(secret_0), S2N_HMAC_SHA384 };
+
+            struct s2n_psk_identity second_identity = { identity_1, sizeof(identity_1) };
+            struct s2n_pre_shared_key second_psk = { second_identity, secret_1, sizeof(secret_1), S2N_HMAC_SHA256 };
+
+            /* One psk */
```

Comment:
You might want to verify that obfuscated_ticket_age is 0, as that's required for external PSKs